framework/telemetry/internal/telemetryconfiguration.h

```diff
@@ -0,0 +1,39 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_TELEMETRY_TELEMETRYCONFIGURATION_H
+#define MU_TELEMETRY_TELEMETRYCONFIGURATION_H
+
+#include "../itelemetryconfiguration.h"
+
+namespace mu::telemetry {
+class TelemetryConfiguration : public ITelemetryConfiguration
+{
+public:
+    TelemetryConfiguration() = default;
```

Comment:
This constructor will be generated automatically

Reply:
I prefer explicit over implicit.